addons/binding/org.openhab.binding.network/README.md

```diff
@@ -179,3 +183,12 @@ sitemap demo label="Main Menu"
 	}
 }
 ```
+
+### Mobile phones
+The following examples can be used to detect the presence mobile phones in your network. Because of the complex power saving mechanisms in mobile phone operating systems, the parameters given in these examples might need tuning for reliable detection of the devices. Especially for iPhones, which have aggressive power saving mechanisms. They are intended to serve as a starting point:
```

Comment:
With the default parameters (retry=1, timeout=5000, refreshInterval=60000) one will get unreliable presence detection for mobile devices.

I played with the parameters and found the suggested ones to deliver continuous presence signals, at the cost of some latency on departure.

In my setup the iPhone is much more aggressive at saving power compared to the Android 8 device. This might be different with Android 9, such a device is not available to me.

Why not make it easy for users and give them some hints? Especially considering that you need to test at least a few hours with a new set of parameters.

Reply:
If you can condense the paragraph to maximum two lines and add it as upfront comments to an example  textual Thing, that would be sweet. 

The text is already long, and this is really best shown as config example in my opinion :)